helpers/remote_rooms/remote_rooms_location.py

```diff
@@ -0,0 +1,11 @@
+
+def map_remote_room_location_to_filter():
+    '''
+    switcher for mapping remote room locations
```

Comment:
Good work @verenceLola, This docstring can be more descriptive. Consider including return type. Ref: [Python3 Docs](https://www.datacamp.com/community/tutorials/docstrings-python)

Reply:
Thank you for the material @ThaDeveloper